### PR TITLE
GH-6: update npm dependencies

### PR DESCRIPTION
**Pull request by alex-waverley**
There were updated npm dependencies in order to avoid sequrity vulnerabillities in this PR . As a result frontend part was moved to the latest version of CRA, which is not ejected. It means, that we don't need to manitain a lot of dependencies related to CRA by our hands.

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -3,85 +3,37 @@
...
-    "babel-jest": "^24.9.0",
-    "babel-loader": "8.0.6",
-    "babel-plugin-named-asset-import": "^0.3.4",
-    "babel-preset-react-app": "^9.0.2",
-    "camelcase": "^5.2.0",
-    "case-sensitive-paths-webpack-plugin": "2.2.0",
-    "classnames": "^2.2.6",
-    "css-loader": "2.1.1",
-    "customize-cra": "^0.8.0",
-    "dotenv": "6.2.0",
```
Comment (sskorol) [inline frontend/package.json:27]: Check required peer deps, please.
Seems like `dotenv` and a couple of others are required.
At least I could start the app in docker after getting these warns.

File: frontend/public/index.html [hunk 2]
```diff
@@ -2,8 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```
Comment (sskorol) [inline frontend/public/index.html:5]: I see the previous icon was removed. Let's keep it for now until we get new designs.

**Review by sskorol — CHANGES_REQUESTED** [review]

File: frontend/package.json (hunk 1 above)
Comment (alex-waverley) [reply frontend/package.json:27]: fixed by means of updating config-overwrite script

File: frontend/public/index.html (hunk 2 above)
Comment (alex-waverley) [reply frontend/public/index.html:5]: done

**Review by sskorol — APPROVED** [review]

**Pull request merged**